### ISSUE TITLE
Release aexpect 1.7.1

### ISSUE THREAD

**Issue by pevogam** [avocado-framework/aexpect#127]
Since we have done some separation of functionality with Avocado VT and it has been tagged and released as 103.0 perhaps it would be a good idea to also release axpect now for easier traceability  and to provide any of collected changes to the users.

@ldoktor @lmr What do you think about this suggestion?

**Comment by vfazio** [comment]
This would be useful for us since the usage of `pipes` was dropped and thus implicit support of Python 3.13

**Comment by ldoktor** [comment]
Makes sense to me although I'd like to see https://github.com/avocado-framework/aexpect/pull/135 and https://github.com/avocado-framework/aexpect/pull/141 and https://github.com/avocado-framework/aexpect/pull/142 in the release, what do you think?
